dashboard/src/modules/components/AuthComponent/LoginForm.jsx

```diff
@@ -94,14 +101,21 @@ const LoginForm = () => {
             />
           </FormGroup>
           <FormGroup label="Password" isRequired fieldId="password">
-            <TextInput
-              isRequired
-              type="password"
-              id="password"
-              name="password"
-              value={details.password}
-              onChange={handlePasswordChange}
-            />
+            <div className="password-holder">
+              <TextInput
+                isRequired
+                type={showPassword ? "text" : "password"}
+                id="password"
+                name="password"
+                value={details.password}
+                onChange={handlePasswordChange}
+              />
+              <Button
+                variant="control"
+                onClick={onShowPassword}
+                icon={showPassword ? <EyeIcon /> : <EyeSlashIcon />}
```

Comment:
Realistically, either icon convention always leaves me confused. Either you're confirming the *current* state of the toggle or advertising the state you'll *achieve* by clicking on it. I see both conventions scattered through various UIs, which means it's often not clear which convention is used.

This case is less ambiguous than most, as you either see the password characters or you don't, and regardless of the icon it's fairly obvious that clicking on it will result in the opposite behavior.

Most importantly, though, I'd say that we should ensure that throughout the Pbench dashboard we strictly follow a single convention.

Reply:
> Please let's not link PRs to Epics or Features in Jira! We should be linking PRs to stories (or bugs) where they're simple, or to sub-tasks where a story/bug requires multiple PRs.
> 
> This seems to be several bug/story issues under the PBENCH-86 epic.
> 
> It's not necessarily bad to link a PR to multiple simple issues, as long as they're closely related, but in general small focused PRs dealing with a single issue are easier to review.

Noted!

Reply:
Even I have the same confusion. Now I changed the behaviour referring to [`Patternfly Login Page`](https://www.patternfly.org/v4/components/login-page/react/showhide-password/)